ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -101,16 +90,22 @@ export default function TokenList({
         filterCallback: 'inclusive',
       },
     ]);
-    const filteredSortedAssets = sortAssets(filteredAssets, tokenSortConfig);
-    return filteredSortedAssets;
+
+    const nativeTokens = filteredAssets.filter((token) => token.isNative);
+    const nonNativeTokens = filteredAssets.filter((token) => !token.isNative);
+
+    const sortedNativeTokens = sortAssets(nativeTokens, tokenSortConfig);
+    const sortedNonNativeTokens = sortAssets(nonNativeTokens, tokenSortConfig);
+
+    return [...sortedNativeTokens, ...sortedNonNativeTokens];
```

Comment:
Unsure if this is just a thing you're doing to make development easier, but I think that we're probably going to need to ultimately consolidate this into a single `sortAssets` call, so that the native tokens don't always appear at the top of the list

```suggestion
    const sortedTokens = sortAssets([...nativeTokens, ...nonNativeTokens], tokenSortConfig);

    return sortedTokens;
```

(☝️ untested) There might be a cleaner way to do this

Reply:
I thought we always wanted to keep native tokens at the top? I can implement this though

Reply:
I checked with product bc I thought this was weird as well, and the desicions was that `nativeTokens` should be included in the list with the other erc20s.

I anticipate that we may eventually need to implement a "pin token to top" feature down the line, but for now, yes they should be included in the main list.